imap_processing/cdf/utils.py

```diff
@@ -41,6 +42,24 @@ def calc_start_time(shcoarse_time: float) -> np.datetime64:
     return launch_time + time_delta
 
 
+def load_cdf(file_path: Path, **kwargs) -> xr.Dataset:
+    """Load the contents of a CDF file into an ``xarray`` dataset.
+
+    Parameters
+    ----------
+    file_path : Path
+        The path to the CDF file
+    kwargs : dict
+        Keyword arguments for ``cdf_to_xarray``
```

Comment:
https://numpydoc.readthedocs.io/en/latest/format.html
> When documenting variable length positional, or keyword arguments, leave the leading star(s) in front of the name:
```
*args : tuple
    Additional arguments should be passed as keyword arguments
**kwargs : dict, optional
    Extra arguments to `metric`: refer to each metric documentation for a
    list of all possible arguments.
```

```suggestion
    **kwargs : dict, optional
        Keyword arguments for ``cdf_to_xarray``
```

Reply:
Good to know!